aicssegmentation/structure_wrapper/WorkflowStep.py

```diff
@@ -0,0 +1,40 @@
+import importlib
+import numpy as np
+
+class WorkflowStep:
+    """
+    A class that defines a step in an AICS-Segmentation workflow.
+    """
+    def __init__(self, step_config: dict):
```

Comment:
Correct type annotation for a list is `Dict` (from `typing.Dict`). Lowercase `dict` is a function that creates a Dictionary.